templates/vdc/vdc.py

```diff
@@ -170,64 +176,37 @@ def disable(self):
         space.disable('The space should be disabled.')
         self.data['disabled'] = True
 
+    def update_data(self, data):
+        acc = self.account
 
-def get_user_accessright(username, service):
-    for u in service.model.data.uservdc:
-        if u.name == username:
-            return u.accesstype
+        # Get given space, raise error if not found
+        space = acc.space_get(
+            name=self.name,
+            location=self.data['location'],
+            create=False
+        )
 
+        self.data.update(data)
 
-def processChange(job):
-    service = job.service
+        if 'users' in data:
```

Comment:
don't allow user to be changed from `update_data`

Reply:
Yeah, I got it :D